src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +45,10 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("from Movie").getResultList();
+        } catch (Exception e) {
+            throw new RuntimeException("Error getting all Movie entities", e);
```

Comment:
The exception type should be consistent with other methods in the DAO implementation. Consider using `DataProcessingException` instead of `RuntimeException` to handle errors when getting all Movie entities.